README.md

```diff
@@ -2,6 +2,7 @@
 
 > Ark wallet application for Ledger Blue and Nano S
 
+[![Build](https://github.com/ArkEcosystem/ledger/workflows/Build/badge.svg)](https://github.com/ArkEcosystem/ledger/actions)
```

Comment:
```suggestion
[![Build](https://badgen.net/github/status/arkecosystem/ledger)](https://github.com/ArkEcosystem/ledger/actions)
```

It might be unnecessary for this repo since there is only one test suite, but badgen offers an endpoint for the combined status.

Reply:
That's not a bad idea.
It's not necessary ATM though as we don't even have unit tests here yet.
I think it's something to consider later though.

Thanks for the suggestion, @dated 👍

Reply:
Yea, it was more of a general suggestion since the other repos will have to move away from the circleci badge, so to keep things uniform across the board